.pre-commit-config.yaml

```diff
@@ -16,10 +16,13 @@ repos:
         args: [--fix, --exit-non-zero-on-fix]
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.5.1
+    rev: v3.0.1
     hooks:
       - id: prettier
         types_or: [css, scss, javascript, ts, tsx, json, yaml]
+        additional_dependencies:
+          # Keep in sync with package.json
+          - prettier@3.0.1
```

Comment:
Thanks. I wonder if there is a way to reference our package json instead. 

I didn't know this had to be managed here also sorry, I'll do a review tomorrow if no one else gets to it before me.

Reply:
Not that I am aware of, unfortunately!

I've resorted to keeping the two in sync for my own projects, would love to hear if there is a better way.

Reply:
in theory you could make use of `repo: local` - https://pre-commit.com/#repository-local-hooks and use it something like: https://github.com/pre-commit/pre-commit/issues/945#issuecomment-527603460 ([`npm run lint:format`](https://github.com/wagtail/wagtail/blob/main/package.json#L140))

Reply:
https://github.com/wagtail/wagtail/issues/10755 raised.